File: desktop/src/net/namekdev/memcop/desktop/DesktopLauncher.java
package net.namekdev.memcop.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import net.namekdev.memcop.MemcopGame;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		int pixelPerInch = java.awt.Toolkit.getDefaultToolkit().getScreenResolution();
		float scale = pixelPerInch / 96f;
		config.width = (int) (scale * MemcopGame.WIDTH);
		config.height = (int) (scale * MemcopGame.HEIGHT);
		new LwjglApplication(new MemcopGame(), config);
	}
}

File: desktop/src/net/namekdev/memcop/desktop/DesktopLauncher.kt
package net.namekdev.memcop.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import net.namekdev.memcop.MemcopGame
import net.namekdev.memcop.view.Render

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val pixelPerInch = java.awt.Toolkit.getDefaultToolkit().screenResolution
        Render.scale = pixelPerInch / 96f

        val config = LwjglApplicationConfiguration()
        config.width = (Render.width).toInt()
        config.height = (Render.height).toInt()

        LwjglApplication(MemcopGame(), config)
    }
}
